api/applications/applications_handler.go

```diff
@@ -118,6 +120,7 @@ func (ah ApplicationHandler) RegenerateMachineUserToken(appName string) (*applic
 		case <-queryInterval:
 			machineUser, err := ah.getMachineUserForApp(appName)
```

Comment:
Should we also log the error returned by getMachineUserForApp? I don't that this error is logged within the getMachineUserForApp function.